pom.xml

```diff
@@ -1006,6 +1010,10 @@
               <goal>integration-test</goal>
               <goal>verify</goal>
             </goals>
+            <configuration>
+              <excludeGroups>${accumulo.integrationTestsExcludedGroups}</excludeGroups>
+              <groups>${accumulo.integrationTestsGroups}</groups>
+            </configuration>
```

Comment:
> It's very likely we will have more categories soon

... we will? I don't remember anyone talking about wanting to do this. Updating the docs is going to be required no matter what.